.github/workflows/main.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       with:
         repository: nlnetlabs/rpki-deploy
         path: rpki-deploy
+        ref: externalize-tests
```

Comment:
This change is just temporarily to test against the as yet unmerged branch in the rpki-deploy repo. This line should be removed before merging this PR.